### PR TITLE
os/fs: Fix spelling/grammatical mistakes

### PR DESCRIPTION
**Pull request by amogh-samsung**
- Fix spelling mistakes at a few places.
- Update comment indentation for better grammatical understanding.

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@amogh-samsung no conflict with PR #5603 ?

**Review by bobo2108 — DISMISSED** [review]
LGTM!

File: os/fs/smartfs/smartfs.h [hunk 1]
```diff
@@ -175,7 +175,7 @@
 #define INODE_STATE_FILE          (CONFIG_NXFFS_ERASEDSTATE ^ 0x22)
 #define INODE_STATE_DELETED       (CONFIG_NXFFS_ERASEDSTATE ^ 0xaa)
 
-/* Smartfs worbuffer maxuimum length */
+/* Smartfs worbuffer maximum length */
```
Comment (abhishek-samsung) [inline os/fs/smartfs/smartfs.h:178]: is it supposed to be workbuffer instead of worbuffer?

Comment (amogh-samsung) [reply os/fs/smartfs/smartfs.h]: Agreed and updated,
Thanks

File: os/fs/smartfs/smartfs_smart.c [hunk 2]
```diff
@@ -1259,7 +1259,7 @@ static int smartfs_bind(FAR struct inode *blkdriver, const void *data, void **ha
 	}
 
 	/* Initialize the allocated mountpt state structure.  The filesystem is
-	 * responsible for one reference ont the blkdriver inode and does not
+	 * responsible for one reference on the blkdriver inode and does not
```
Comment (sangam-swami) [inline os/fs/smartfs/smartfs_smart.c:1262]: You can add below change as well at line:1249
/* If the global semaphore hasn't been initialized, then
	 * initialized it now. */  => initialize

**Review by thapav — DISMISSED** [review]
LGTM

**Comment by amogh-samsung** [comment]
> Target : [[d88331e](https://github.com/Samsung/TizenRT/commit/d88331e786789516ece804d0d46ad4ba559bc18a)] - Code Rule Check OK.

Hello, sorry for the late reply.
Common changes with #5603 will be removed from 5603 since they do not fall in the scope of that PR directly.
Hence, this PR is raised separately.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**